### ISSUE TITLE
Dynamic Linq with Where and ObjectParameter doesn't work

### ISSUE THREAD

**Issue by Tiku123** [kahanu/System.Linq.Dynamic#15]
I'm using System.Linq.Dynamic I have this code :
..........................................................................................
Imports System.Linq.Dynamic
Dim mylist As IQueryable(Of Myobject)
Dim ob1 As New List(Of ObjectParameter)
Dim dynstr as string=string.empty
 dynstr += "id= @id"
ob1.Add(New ObjectParameter("id", 26))
mylist=context.MyObjects.Where(dynstr,ob1.ToArray()).Tolist
..................................................................................................
I'm expecting that MyList contain only 1 object with id=26 , but MyList contains all the objects inside MyObjects. So it seems that the Where clause has no effect.
  
 It's strange because if I change the dynstr to : 
dynstr+="id=26"
everything works ok. so it seems that the dynstr string "doesn't communicate" with objectParameter list inside the where clause.
  
 What can I do ? Thank you !

**Comment by kahanu** [comment]
@Tiku123 - you would probably get more help if you posted this question on stackoverflow.com.  I have just provided this library for use in NuGet and doing some simple maintenance, but I'm not the author of this package.

It's been years since I've worked in VB.Net so I can't tell if there is an error in your syntax for the "dynstr" variable, or if this is in fact an issue with the ObjectParameter class.  Looking at your code, inside your Where clause, are you sure the ob1.ToArray() is correct?  Have you tried simply using "ob1"?

Sorry I couldn't be more help.
